src/app/fdctl/config/default.toml

```diff
@@ -981,7 +981,7 @@ dynamic_port_range = "8900-9000"
         # genesis, but it might be immediately overriden at runtime if
         # the related features which increase this value are enabled.
         # The features are like update_hashes_per_tick2.
-        hashes_per_tick = 1
+        hashes_per_tick = 12500
```

Comment:
Let's leave this for another change. Needs more thought what the default should be (this prevents early epoch rollovers in development).